src/app/spaces/[id]/deliberation/_components/poll.tsx

```
@@ -0,0 +1,7 @@
+'use client';
+
+import React from 'react';
+
+export default function PollPage({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)